Makefile

```diff
@@ -67,19 +67,23 @@ mock_gen:
 # Unit tests with faster execution (nicer for development).
 .PHONY: fast-test
 fast-test:
-	go test -mod=vendor -race ./...
+	go test -mod=vendor -race $(shell go list ./... | grep -v e2e)
```

Comment:
do you know if this would exclude any file with e2e in the name?  Could we make this more specific to files within the e2e/ directory

Reply:
It would, yes. Updated the pattern to match `/e2e$` and `/e2e/.*` instead.